packages/renderer/src/lib/preferences/PreferencesKubernetesContextsRendering.svelte

```diff
@@ -243,14 +257,25 @@ async function connect(contextName: string): Promise<void> {
         <div class="grow flex-column divide-gray-900 text-[var(--pd-invert-content-card-text)]">
           <div class="flex flex-row">
             <div class="flex-none w-36">
-              {#if context.isReachable}
+              {#if context.isReachable || context.isOffline}
```

Comment:
nit: I think:
```
{#if context.isOffline}
...
{:else if context.isReachable}
...
{/if}
```
would be more readable than the embedded if.

Reply:
But there are 30 lines in common, after the embedded if